### ISSUE TITLE
Upstream merge request.

### ISSUE THREAD

**Issue by mibby** [PaperMC/Paper#1000]
Netty version was bumped.

**Comment by aikar** [comment]
I'll try tonight, but is there a specific known fix in this netty update to make it special?

Generic dep version updates are not a good reason for bumping upstream priority.

But the update has some conflicts with my patches so I gotta find time to look into it.

**Comment by mibby** [comment]
Nothing serious in particular as far as I know. I just like to keep netty versions consistent between Waterfall and Paper, as well as stay up to date with upstream Spigot. :)

Hopefully the Hopper patch could also be merged into Paper since it seems to be working wonders in performance for me too!

**Comment by electronicboy** [comment]
I've done the merge locally, had to rework the layout, waiting on @aikar to take a quick look at it/do his own stab at it; I'm not amazingly happy with how it looks, but not entirely sure how I'd clean it up any further

**Comment by electronicboy** [comment]
Done, https://ci.destroystokyo.com/job/Paper/1323/
